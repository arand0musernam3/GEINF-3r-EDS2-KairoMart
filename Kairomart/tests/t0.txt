../data
5
1
jugador_maniqui
5
6
1
4
3
