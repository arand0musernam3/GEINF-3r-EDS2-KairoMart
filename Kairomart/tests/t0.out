Submit the initial data path:
Menú principal:
1. Introduir jugador
2. Moure vehicle
3. Executar tick
4. Mostrar situació de la carrera
5. Para l'àudio, si us plau!
0. Sortir
Menú principal:
1. Introduir jugador
2. Moure vehicle
3. Executar tick
4. Mostrar situació de la carrera
0. Sortir
Entra el nom del jugador
SELECCIONAT // 1. willy: The fear of girls
SELECCIONAT // 2. nani: The fear of watatata
SELECCIONAT // 3. jordiconda: The fear of fat
SELECCIONAT // 4. pafefo: The fear of Sergi Roberto
SELECCIONAT // 5. uri_sis: The fear of Uri
6. kairo: The fear of KairoMart
7. toast: The fear of Mushrooms
8. paperdewater: The fear of Frxas
Entra L'ÍNDEX del personatge (que no s'hagi escollit)
SELECCIONAT // 1. willy: The fear of girls
SELECCIONAT // 2. nani: The fear of watatata
SELECCIONAT // 3. jordiconda: The fear of fat
SELECCIONAT // 4. pafefo: The fear of Sergi Roberto
SELECCIONAT // 5. uri_sis: The fear of Uri
6. kairo: The fear of KairoMart
7. toast: The fear of Mushrooms
8. paperdewater: The fear of Frxas
Entra L'ÍNDEX del personatge (que no s'hagi escollit)
1. Quadriga
2. Biga
3. Horse
Entra L'ÍNDEX del vehicle
Jugadors inscrits:
1. greixina (willy, Quadriga): 0.0m/s 0.0rad 0.0m(0%)
2. naniol (nani, Biga): 0.0m/s 0.0rad 0.0m(0%)
3. lil_wiltxer (jordiconda, Horse): 0.0m/s 0.0rad 0.0m(0%)
4. diabla_de_tres_colores (pafefo, Quadriga): 0.0m/s 0.0rad 0.0m(0%)
5. uriol (uri_sis, Biga): 0.0m/s 0.0rad 0.0m(0%)
6. jugador_maniqui (kairo, Quadriga): 0.0m/s 0.0rad 0.0m(0%)
Menú principal:
1. Introduir jugador
2. Moure vehicle
3. Executar tick
4. Mostrar situació de la carrera
0. Sortir
1. greixina (willy, Quadriga): 0.0m/s 0.0rad 0.0m(0%)
2. naniol (nani, Biga): 0.0m/s 0.0rad 0.0m(0%)
3. lil_wiltxer (jordiconda, Horse): 0.0m/s 0.0rad 0.0m(0%)
4. diabla_de_tres_colores (pafefo, Quadriga): 0.0m/s 0.0rad 0.0m(0%)
5. uriol (uri_sis, Biga): 0.0m/s 0.0rad 0.0m(0%)
6. jugador_maniqui (kairo, Quadriga): 0.0m/s 0.0rad 0.0m(0%)
Menú principal:
1. Introduir jugador
2. Moure vehicle
3. Executar tick
4. Mostrar situació de la carrera
0. Sortir
1. jugador_maniqui (kairo, Quadriga): 0.0m/s 0.0rad 0.0m(0%)
2. uriol (uri_sis, Biga): 0.0m/s 0.0rad 0.0m(0%)
3. diabla_de_tres_colores (pafefo, Quadriga): 0.0m/s 0.0rad 0.0m(0%)
4. lil_wiltxer (jordiconda, Horse): 0.0m/s 0.0rad 0.0m(0%)
5. naniol (nani, Biga): 0.0m/s 0.0rad 0.0m(0%)
6. greixina (willy, Quadriga): 0.0m/s 0.0rad 0.0m(0%)
Menú principal:
1. Introduir jugador
2. Moure vehicle
3. Executar tick
4. Mostrar situació de la carrera
0. Sortir

Process finished with exit code 0
